src/plugins/util/web.js

```diff
@@ -69,6 +69,10 @@ const web = {
     wrapEnd(req)
     wrapEvents(req)
 
+    if (process.env.SIGNALFX_SERVER_TIMING_CONTEXT === 'true') {
```

Comment:
I don't remember how we handle reading env vars in the rest of the repo but do you think it's worth to lowercase and trim extra whitespace before comparing to `"true"`?

Reply:
I don't believe there's string to boolean helper used by the config utils and `someVar === "true"` is the current practice.

Reply:
I just meant lower-casing and trimming white-space, not casting to bool. People might use `True` or `TRUE` instead of `true`, or more likely might accidentally add trailing or leading whitespace around the string.

Reply:
Yep, I'll make this better.